python/taichi/lang/ast/ast_transformer.py

```diff
@@ -329,34 +334,59 @@ def build_Call(ctx, node):
             else:
                 args.append(arg.ptr)
         keywords = dict(ChainMap(*[keyword.ptr for keyword in node.keywords]))
-
-        if isinstance(node.func, ast.Attribute):
-            attr_name = node.func.attr
-            if attr_name == 'format' and isinstance(node.func.value.ptr, str):
-                args.insert(0, node.func.value.ptr)
-                node.ptr = impl.ti_format(*args, **keywords)
-            else:
-                node.ptr = node.func.ptr(*args, **keywords)
-        elif isinstance(node.func, ast.Name):
-            func_name = node.func.id
-            if func_name == 'print':
-                node.ptr = impl.ti_print(*args, **keywords)
-            elif func_name == 'min':
-                node.ptr = ti_ops.ti_min(*args, **keywords)
-            elif func_name == 'max':
-                node.ptr = ti_ops.ti_max(*args, **keywords)
-            elif func_name == 'int':
-                node.ptr = impl.ti_int(*args, **keywords)
-            elif func_name == 'float':
-                node.ptr = impl.ti_float(*args, **keywords)
-            elif func_name == 'any':
-                node.ptr = ti_ops.ti_any(*args, **keywords)
-            elif func_name == 'all':
-                node.ptr = ti_ops.ti_all(*args, **keywords)
-            else:
-                node.ptr = node.func.ptr(*args, **keywords)
+        func = node.func.ptr
+
+        if isinstance(node.func, ast.Attribute) and isinstance(
+                node.func.value.ptr, str) and node.func.attr == 'format':
+            args.insert(0, node.func.value.ptr)
+            node.ptr = impl.ti_format(*args, **keywords)
```

Comment:
Can we make this more organized by having a `build_builtin` or something similar? 
Main goal is to know where to find supported builtin functions in one place.